build.gradle.kts

```diff
@@ -184,14 +184,19 @@ subprojects {
         val isLocalizationsProject = project.name == "localization"
         lintConfig = rootProject.file(if (isLocalizationsProject) "lint-no-op.xml" else "lint.xml")
         xmlReport = true
+        sarifReport = true
+        abortOnError = false
```

Comment:
for the context: I've added `abortOnError = false` because of this: https://buildkite.com/automattic/pocket-casts-android/builds/8293#0191e554-9473-4617-914b-60b0f2cc2f85/179-271 : after failed lint task, the `sarif` file for `app` was not generated (because `automotive:lint` execution stopped the task)

Maybe we should run `:app:lint` as the first task then, and only then `:automotive` and `:wear`

Reply:
Maybe we could 2 separate jobs? One to run `:app:lint` for the GH integration. And another one for the other apps.

Reply:
Probably addressed in: 6de6674